asciidoctorj-core/src/main/resources/org/asciidoctor/internal/asciidoctorclass.rb

```diff
@@ -19,50 +21,54 @@ def unregister_all_extensions()
         Asciidoctor::Extensions.unregister_all
     end
 
-    def docinfo_processor(extensionName)
-        Asciidoctor::Extensions.register do
+    def unregister_extension name
+        Asciidoctor::Extensions.unregister name
+    end
+
+    def docinfo_processor(extensionName, groupName = nil)
+        Asciidoctor::Extensions.register (groupName != nil ? groupName.to_sym : nil) do
```

Comment:
In that case, you can write it as:

```
Asciidoctor::Extensions.register(groupName && groupName.to_sym)
```

or

```
Asciidoctor::Extensions.register(groupName ? groupName.to_sym : nil)
```

A method argument list is an implicit grouping expression.